cmd/cycles.go

```diff
@@ -33,7 +33,7 @@ var cyclesCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		depGraph, _, mainModule := getDepInfo()
 		var cycleChains [][]string
-		chains := make(map[int][]string)
+		chains := make(map[int][][]string)
```

Comment:
I am starting to feel like these types are becoming "too much". Can we introduce some structs at least for one level of the arrays?

Actually I am checking the code now, why are we even storing the chains in map and what is that map used for? Looks like we're only writing to it and never reading from it? https://github.com/kubernetes-sigs/depstat/blob/ae160347164e78bd3108c699b45886b25ef5b6b2/cmd/utils.go#L51

Reply:
Thanks for the review @alenkacz!

Yeah, I totally forgot about using structs 😅 I'll update the PR replacing `map[int][][]string` with a struct.

To answer your other question I think we will still need maps since once we have the length of the longest chain/s, maps make it easier to retrieve the actual chain/s like done [here](https://github.com/kubernetes-sigs/depstat/pull/28/files#diff-fac57bffaaff7a817f0e2324e687502eff4ee3227ccf83f4a49f673573978c22R68-R71). 

The way to do this without using maps would be to iterate the `[][]string` and print each `[]string` which has a length equal to "the length of the longest chain/s". Maps felt a bit more convenient to me since now instead of printing a single longest chain we print **all** the longest chains.

If there is a better way of doing this please let me know 🙂

Reply:
if you only need the longest one you can always have struct like:
```
type Chains struct {
 chains []Chain

 longestChainsLength int
 longestChains []Chain
}
```

then we could have something like
```
type Chain struct {
 length int
 chain []string
}
```

Just a quick idea. WDYT?

Reply:
This looks perfect! Thanks for this. Will update the PR tomorrow :)